static/src/javascripts/projects/common/views/weather.html

```diff
@@ -1,28 +1,30 @@
-    <ul class="u-unstyled weather__forecasts">
-        <li class="weather__forecast">
-            <p class="weather__desc">
-                <span class="u-h">The temperature</span>
-                <span class="weather__time">now</span>
-                <span class="u-h">is</span>
-                <span class="weather__temp js-weather-temp">{{tempNow}}</span>
-            </p>
-            <i class="i i-weather-{{icon}} weather__icon js-weather-icon" title="{{description}}"><span class="u-h">{{description}}</span></i>
-        </li>
-    </ul>
+    <div class="u-unstyled weather__current">
+        <p class="weather__desc">
+            <span class="u-h">The temperature</span>
+            <span class="weather__time">now</span>
+            <span class="u-h">is</span>
+            <span class="weather__temp js-weather-temp">{{tempNow}}</span>
+        </p>
+        <i class="i i-weather-{{icon}} weather__icon js-weather-icon" title="{{description}}"><span class="u-h">{{description}}</span></i>
+    </div>
+    <a class="weather__toggle-forecast js-toggle-forecast" data-link-name="weather-toggle-forecast" data-toggle-class="js-weather-forecasts-mobile"
+        href="#toggle-forecast"><span class="u-h">Toggle forecast</span><i class="i i-arrow-grey-large weather__toggle-icon"></i>
+    </a>
+    <ul class="js-weather-forecasts weather__forecasts weather__forecasts--desktop hide-until-leftcol"></ul>
     <div class="weather__location">
         <div class="search-tool js-search-tool">
             <div class="search-tool__form u-cf">
-                <input class="search-tool__input js-search-tool-input js-weather-input" type="text" data-link-name="weather-list-location" value="{{location}}" />
+                <label class="u-h" for="editlocation">Edit your location</label>
+                <input id="editlocation" class="search-tool__input js-search-tool-input js-weather-input" type="text" data-link-name="weather-edit-location" value="{{location}}" />
+                <i class="weather__edit-location i i-search-dark-grey-36 weather__search-icon"></i>
                 <button class="search-tool__btn"><i class="i i-search-white-36"></i><span class="u-h">Search for city</span></button>
             </div>
             <ul class="u-unstyled search-tool__list js-search-tool-list"></ul>
         </div>
-        <a class="weather__edit-location js-edit-location" data-link-name="weather-edit-location"
-            href="#edit-location"><span class="u-h">Edit my location</span><i class="i i-search-dark-grey-36 weather__search-icon"></i>
-        </a>
         <a class="weather__close-location js-close-location" data-link-name="weather-close-location"
             href="#close-change-location"><span class="u-h">Close change location</span><i class="i i-close-icon-grey weather__close-icon"></i>
         </a>
     </div>
 </div>
+<ul class="js-weather-forecasts js-weather-forecasts-mobile weather__forecasts weather__forecasts--mobile u-h"></ul>
```

Comment:
This is a gigantic template to be included in the JavaScript file, which gets loaded on every page. I'd prefer it if this just lived in the fronts initial payload, or ajaxed in. @gklopper and I have been discussing the trade-offs with this recently. Happy to chat more about it in person.